l10n_it_sepa_bonifici/README.rst

```diff
@@ -0,0 +1,106 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+====================================
+Italian Localization - SEPA Bonifici
+====================================
+
+This module adds a new payment export types to use in the payment order.
+For italian credit transfer, the format code is CBIBdyPaymentRequest.00.04.00
+
+Note (ITA):
+
+le specifiche CBI del bonifico XML SEPA (versione 00.03.09) si basano
+sul messaggio ISO20022 pain.001.001.03 e sono compliant al Rulebook SEPA.
+Come su riportato, il messaggio pain.001.001.03 non è stato utilizzato
+integralmente, essendo  strutturato in maniera tale da poter essere applicabile
+ad una pletora molto estesa di casistiche e di soggetti.
+Ne consegue che, al fine di implementare le regole di comunità, il tracciato
+SEPA Credit Transfer CBI preso ad esempio è un sottoinsieme del succitato
+messaggio ISO e raccoglie i requisiti necessari alla corretta esecuzione di un
+bonifico in Italia, come ad esempio la presenza obbligatoria dell’ABI della
+banca di addebito contenuto nel campo “MmbId” che è invece facoltativo
+nel tracciato ISO. 
+
+Configuration
+=============
+
+Configure the CUC code and the Issuer code (must be "CBI").
+
+.. image:: /l10n_it_sepa_bonifici/static/company_setting.png
```

Comment:
Please use ```.. figure::``` to also support apps.odoo.com
See https://raw.githubusercontent.com/OCA/account-financial-reporting/9.0/mis_builder/README.rst